src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {

```

Comment:
The `ExcavatorProducer` class should parameterize the `MachineProducer` interface with the `Excavator` type. According to the task requirements, you should replace `Object` in `get()` with the suitable option, such as `List<Excavator> get();`.